src/qililab/instruments/quantum_machines/qmm.py

```diff
@@ -0,0 +1,159 @@
+# Copyright 2023 Qilimanjaro Quantum Tech
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Copyright 2023 Qilimanjaro Quantum Tech
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Quantum Machines OPX class."""
+from dataclasses import dataclass
+from typing import Dict
+
+from qm import QuantumMachine, QuantumMachinesManager, SimulationConfig
+from qm.jobs.running_qm_job import RunningQmJob
+from qm.qua import Program
+
+from qililab.instruments.instrument import Instrument
+from qililab.instruments.utils import InstrumentFactory
+from qililab.result.quantum_machines_results import QuantumMachinesResult
+from qililab.typings import InstrumentName, Parameter, QMMDriver
+
+
+@InstrumentFactory.register
+class QMM(Instrument):
```

Comment:
Could we rename this to a full descriptive name? Like `QuantumMachinesManager` or `QuantumMachinesInstrument` or just `QuantumMachines`?

Reply:
done

Reply:
Done